.gitattributes

```
@@ -0,0 +1 @@
+Sources/JavaScriptKit/Runtime linguist-generated
```

```suggestion
Sources/JavaScriptKit/Runtime/* linguist-generated
```

According to https://git-scm.com/docs/gitattributes
> patterns that match a directory do not recursively match paths inside that directory (so using the trailing-slash path/ syntax is pointless in an attributes file; use path/** instead)